doc/conf.py

```diff
@@ -345,6 +345,16 @@
 
 traceability_json_export_path = '_build/exported_items.json'
 
+# {regex: (default, :hover+:active, :visited)}
+# css-file will be built in the same order, which means that the bottom regex has highest priority
```

Comment:
Can we add this into the docs as well?

Reply:
- After investigating more, I found out that insertion-order preservation nature of dict objects is new in Python 3.7. To keep the plugin's behaviour the same across all supported Python versions, we decided to use an `OrderedDict` for deterministic order in py<3.7 as well efdc9723860bb00707143d37b71be4d98a0fb6c2.
- Added new feature to readme b0b452638184ebdfd16ecb9b6ab51d49ade7f845.